video_creation/data/videos.json

```diff
@@ -1 +1,179 @@
-[]
+[
```

Comment:
You don't need to commit your personal `videos.json` file!

Reply:
it is part of the .gitignore. I'm not sure why it came through. it is a good example for the changes thatneed to be made though